src/server/components/lists/types.ts

```diff
@@ -9,6 +9,9 @@ export enum QuestionName {
   "readDisclaimer" = "readDisclaimer",
   "resultsTurnaround" = "resultsTurnaround",
   "readCovidDisclaimer" = "readCovidDisclaimer",
+  "insurance" = "insurance",
+  "contactInsurance" = "contactInsurance",
```

Comment:
quick fix here for the comment I left here `src/server/components/lists/helpers.ts`, is to do 
`enum QuestionName {
  "contactInsurance" = "contact-insurance"
}`

the problem with enums in TS is they're not reflectable like this (i.e. reverse lookup).. I won't get into this rant now lol. what my suggestion does is, in the code, you would refer to the question as `QuestionName.contactInsurance` - but that actually uses the value "contact-insurance" and will do a string match. 

although when you get into that territory, you should really just use a normal object

Reply:
as discussed will cover with a separate refactor ticket